tcms/rpc/tests/test_testcase.py

```diff
@@ -114,6 +119,72 @@ def test_update_author_issue_630(self):
         self.assertEqual(self.new_author, self.testcase.author)
         self.assertEqual(self.new_author.pk, updated['author_id'])
 
+    def test_update_author_should_fail_for_fake_user_id(self):
+        with self.assertRaises(Fault):
+            self.rpc_client.TestCase.update(  # pylint: disable=objects-update-used
+                self.testcase.pk,
+                {
+                    'author': self.fake_user_id,
+                }
+            )
+
+        self.testcase.refresh_from_db()
+        self.assertNotEqual(self.fake_user_id, self.testcase.author.pk)
```

Comment:
a better approach is to record the author_id in a variable at the beginning of the test and then verify that it has not changed.